package.json

```diff
@@ -60,7 +60,8 @@
     "phantom": "^0.8.4",
     "phantomjs-prebuilt": "^2.1.7",
     "vow": "^0.4.10",
-    "vow-node": "^0.3.0"
+    "vow-node": "^0.3.0",
+    "html-entities": "^1.2.0"
```

Comment:
It allows to do one more transformation which is required for several parsers - decodeHTML.
We need to think about ability to specify what actions and transformation you need to build the best match package of goose.